WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Connection Alerts/PointOfSaleCardPresentPaymentBluetoothRequiredAlertView.swift

```diff
@@ -33,7 +33,7 @@ struct PointOfSaleCardPresentPaymentBluetoothRequiredAlertView: View {
 
             Button(viewModel.openSettingsButtonViewModel.title,
                    action: viewModel.openSettingsButtonViewModel.actionHandler)
-            .buttonStyle(POSPrimaryButtonStyle())
+            .buttonStyle(POSButtonStyle(variant: .filled, size: .normal))
```

Comment:
Question/Preference: In the `POSButtonStyle` instances that use variant filled & size normal  we could init them without passing any property (as these are the default values) just so there's less code.

```suggestion
            .buttonStyle(POSButtonStyle())
```

Reply:
Personally, I think having one `POSButtonStyle` makes the variant & size usage more explicit. We could also create wrappers like `POSFilledNormalButtonStyle` and the other 3 combinations, but I feel like it's not much shorter than just specifying `POSButtonStyle` directly. @staskus WDYT? I can update it separately based on the majority opinion.